.gitignore

```diff
@@ -56,4 +56,8 @@ yarn-debug.log*
 yarn-error.log*
 
 # http caches from running kubectl in integration tests
-integration/.kube
\ No newline at end of file
+integration/.kube
+
+### Files owned by indivual developers
+# I want to be able to play with my Tiltfile w/o committing on unrelated projects
+Tiltfile.maia
```

Comment:
have you considered using a global gitignore for this?

Reply:
😳 yes will do